ddsrouter_yaml/test/TestUtils/test_utils.hpp

```diff
@@ -54,19 +54,46 @@ Guid random_guid(
 
 struct TopicInput
 {
+    TopicInput(
```

Comment:
**NOT FOR THIS PR**
We should move these repeated codes to a common place.
I did not do it because moving them to `core` would mean to have code only used for testing installed with the library. However, this could be the best option in long term.

Reply:
Agree